components/webext-storage/README.md

```diff
@@ -0,0 +1,94 @@
+# WebExtension Storage Component
+
+The WebExtension Storage component can be used to power an implementation of the
+[`chrome.storage.sync`](https://developer.chrome.com/extensions/storage) WebExtension API,
+which gives each WebExtensions its own private key-value store that will sync between a user's
+devices. This particular implementation sits atop [Firefox Sync](../sync_manager/README.md).
```

Comment:
minor nit: It might be worth adding a small note saying something like "Note: with a small amount of work, this would also be capable of powering `chrome.storage.local`"?